### PR TITLE
Updating docs for qiskit-transpiler-service

### PR DESCRIPTION
**Pull request by cbjuan**
Updating the documentation to reflect some of the changes we've implemented for the beta version of the service and to include citation reference

### CODE REVIEW

File: docs/transpile/qiskit-transpiler-service.mdx [hunk 1]
```diff
@@ -26,16 +26,17 @@ pip install qiskit-transpiler-service
...
 - `qiskit_transpile_options` (dict) - A Python dictionary object that can include any other option that is valid in the [Qiskit `transpile()` method](defaults-and-configuration-options). If the `qiskit_transpile_options` input includes `optimization_level`, it is discarded in favor of the `optimization_level` specified as parameter input. If the `qiskit_transpile_options` includes any option not recognized by the Qiskit `transpile()` method, the library raises an error.
 
+To get more info about the `qiskit-transpiler-service` available methods, check out the [qiskit-transpiler-service API reference](https://docs.quantum.ibm.com/api/qiskit-transpiler-service). To know more about the service API, check out the [Qiskit Transpiler Service REST API documentation](https://docs.quantum.ibm.com/api/qiskit-transpiler-service-rest)
```
Comment (Eric-Arellano) [inline docs/transpile/qiskit-transpiler-service.mdx]:
```suggestion
To get more info about the `qiskit-transpiler-service` available methods, check out the [qiskit-transpiler-service API reference](/api/qiskit-transpiler-service). To know more about the service API, check out the [Qiskit Transpiler Service REST API documentation](/api/qiskit-transpiler-service-rest)
```

**Review by Eric-Arellano — COMMENTED** [review]
Thanks!

File: docs/transpile/ai-transpiler-passes.mdx [hunk 2]
```diff
@@ -101,3 +101,17 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 - *CollectPermutations*: Collects blocks of `SWAP` circuits as `Permutations`.
 
 These custom collection passes limit the sizes of the collected sub-circuits so that they are supported by the AI synthesis passes, so it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+
+
+## Citation
```
[the remark this answers is not in the document]
Comment (cbjuan) [reply docs/transpile/ai-transpiler-passes.mdx:106]: Thanks. Done in https://github.com/Qiskit/documentation/pull/1435/commits/0283aff35b6a75ec118f072c52c17e2895245fc4

File: docs/transpile/ai-transpiler-passes.mdx [hunk 3]
```diff
@@ -101,3 +101,8 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
...
+
+If you use any AI feature from the Qiskit transpiler service in your research, check out [the recommended citation](./qiskit-transpiler-service#citation).
```
Comment (beckykd) [inline docs/transpile/ai-transpiler-passes.mdx:108]:
```suggestion
If you use any AI feature from the Qiskit transpiler service in your research, use [the recommended citation](./qiskit-transpiler-service#citation).
```

Comment (Eric-Arellano) [reply docs/transpile/ai-transpiler-passes.mdx:108]: > I'm not sure what the tone is supposed to be.

We should be using a consistent tone for all docs

Comment (cbjuan) [reply docs/transpile/ai-transpiler-passes.mdx]: Updated to be more friendly

File: docs/transpile/ai-transpiler-passes.mdx [hunk 4]
```diff
@@ -100,4 +100,9 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 - *CollectLinearFunctions*: Collects blocks of `SWAP` and `CX` as `LinearFunction` objects and stores the original sub-circuit to compare against it after synthesis.
 - *CollectPermutations*: Collects blocks of `SWAP` circuits as `Permutations`.
 
-These custom collection passes limit the sizes of the collected sub-circuits so that they are supported by the AI synthesis passes, so it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+
+
+## Citation
+
+If you use any AI feature from the Qiskit transpiler service in your research, don't forget to use [the recommended citation](./qiskit-transpiler-service#citation)!
```
Comment (Eric-Arellano) [inline docs/transpile/ai-transpiler-passes.mdx:108]: This was a bad suggestion - we want to use a consistent professional tone in our docs

```suggestion
If you use any AI feature from the Qiskit transpiler service in your research, use [the recommended citation](./qiskit-transpiler-service#citation).
```

File: docs/transpile/qiskit-transpiler-service.mdx [hunk 5]
```diff
@@ -68,12 +69,42 @@ The following examples demonstrate how to transpile circuits using the Qiskit tr
...
     ```
 
+3. Produce a similar circuit and transpile it while letting the service to decide if using the AI-powered transpiling passes.
+
+
+    ```python
+    from qiskit.circuit.library import EfficientSU2
+    from qiskit_transpiler_service.transpiler_service import TranspilerService
+
+    circuit = EfficientSU2(101, entanglement="circular", reps=1).decompose()
```
[the remark this answers is not in the document]
Comment (cbjuan) [reply docs/transpile/qiskit-transpiler-service.mdx:85]: I do agree we could do it with any `QuantumCircuit`, yet we want to showcase from the beginning this works for big circuits, that's the reason for that particular one

**Review by Eric-Arellano — APPROVED** [review]

**Comment by Eric-Arellano** [comment]
@cbjuan I've invited you with write access. You can merge this when you're ready, and then I'll deploy the docs.

**Comment by abbycross** [comment]
Sorry - I'm just back from being out sick and want to give this a pass. Will merge asap I promise! I wonder if we should be saying "AI-powered synthesis passes" rather than "AI synthesis passes", per @JulesMurphy 's caution not to leave AI unmodified -- thoughts???

File: docs/transpile/qiskit-transpiler-service.mdx [hunk 6]
```diff
@@ -26,16 +26,17 @@ pip install qiskit-transpiler-service
 
 By default, the package tries to authenticate to IBM Quantum services with the defined Qiskit API token, and uses your token from the `QISKIT_IBM_TOKEN` environment variable or from the file `~/.qiskit/qiskit-ibm.json` (under the section `default-ibm-quantum`).
 
-*Note*: This package requires Qiskit SDK 1.0 by default.
+*Note*: This package requires Qiskit SDK 1.X.
```
Comment (abbycross) [inline docs/transpile/qiskit-transpiler-service.mdx:29]:
```suggestion
*Note*: This package requires Qiskit SDK v1.X.
```

File: docs/transpile/qiskit-transpiler-service.mdx [hunk 7]
```diff
@@ -26,16 +26,17 @@ pip install qiskit-transpiler-service
...
 - `optimization_level` (int) - The potential optimization level to apply during the transpilation process. Valid values are [1,2,3], where 1 is the least optimization (and fastest), and 3 the most optimization (and most time-intensive).
-- `ai` (bool) - Whether to use AI capabilities during transpilation. The AI capabilities available can be for `AIRouting` transpiling passes or other AI synthesis methods. If this value is `True`, the service applies different AI-powered transpiling passes depending on the `optimization_level` requested.
+- `ai` ("true", "false", "auto") - Whether to use AI capabilities during transpilation. The AI capabilities available can be for `AIRouting` transpiling passes or other AI synthesis methods. If this value is `"true"`, the service applies different AI-powered transpiling passes depending on the `optimization_level` requested. If  `"false"`, it uses the latest Qiskit transpiling features without AI. Finally, if `"auto"`, the service decides whether to apply the standard Qiskit heuristic passes or the AI-powered passes based on your circuit.
```
Comment (abbycross) [inline docs/transpile/qiskit-transpiler-service.mdx:36]:
```suggestion
- `ai` ("true", "false", "auto") - Whether to use AI-powered capabilities during transpilation. The AI-powered capabilities available can be for `AIRouting` transpiling passes or other AI synthesis methods. If this value is `"true"`, the service applies different AI-powered transpiling passes depending on the `optimization_level` requested. If  `"false"`, it uses the latest Qiskit transpiling features without AI. Finally, if `"auto"`, the service decides whether to apply the standard Qiskit heuristic passes or the AI-powered passes based on your circuit.
```

File: docs/transpile/ai-transpiler-passes.mdx [hunk 8]
```diff
@@ -100,4 +100,9 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 - *CollectLinearFunctions*: Collects blocks of `SWAP` and `CX` as `LinearFunction` objects and stores the original sub-circuit to compare against it after synthesis.
 - *CollectPermutations*: Collects blocks of `SWAP` circuits as `Permutations`.
 
-These custom collection passes limit the sizes of the collected sub-circuits so that they are supported by the AI synthesis passes, so it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+
+
+## Citation
+
+If you use any AI feature from the Qiskit transpiler service in your research, use [the recommended citation](./qiskit-transpiler-service#citation).
```
Comment (abbycross) [inline docs/transpile/ai-transpiler-passes.mdx:108]:
```suggestion
If you use any AI-powered feature from the Qiskit transpiler service in your research, use [the recommended citation](./qiskit-transpiler-service#citation).
```

File: docs/transpile/qiskit-transpiler-service.mdx [hunk 9]
```diff
@@ -68,12 +69,42 @@ The following examples demonstrate how to transpile circuits using the Qiskit tr
...
+        backend_name="ibm_sherbrooke",
+        ai="auto",
+        optimization_level=1,
+    )
+    transpiled_circuit = cloud_transpiler_service.run(circuit)
+    ```
+
+## Citation
+
+If you use any AI feature from the Qiskit transpiler service in your research, use the following recommended citation:
```
Comment (abbycross) [inline docs/transpile/qiskit-transpiler-service.mdx:97]:
```suggestion
If you use any AI-powered feature from the Qiskit transpiler service in your research, use the following recommended citation:
```

File: docs/transpile/ai-transpiler-passes.mdx [hunk 10]
```diff
@@ -100,4 +100,9 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 - *CollectLinearFunctions*: Collects blocks of `SWAP` and `CX` as `LinearFunction` objects and stores the original sub-circuit to compare against it after synthesis.
 - *CollectPermutations*: Collects blocks of `SWAP` circuits as `Permutations`.
 
-These custom collection passes limit the sizes of the collected sub-circuits so that they are supported by the AI synthesis passes, so it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
```
Comment (abbycross) [inline docs/transpile/ai-transpiler-passes.mdx:103]:
```suggestion
These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI-powered synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
```

**Comment by cbjuan** [comment]
Yes, that's right @abbycross. In general we should say always AI-powered. Thanks for the suggestion and for jumping in right on time :)

File: docs/transpile/qiskit-transpiler-service.mdx [hunk 11]
```diff
@@ -26,16 +26,17 @@ pip install qiskit-transpiler-service
...
 - `optimization_level` (int) - The potential optimization level to apply during the transpilation process. Valid values are [1,2,3], where 1 is the least optimization (and fastest), and 3 the most optimization (and most time-intensive).
-- `ai` (bool) - Whether to use AI capabilities during transpilation. The AI capabilities available can be for `AIRouting` transpiling passes or other AI synthesis methods. If this value is `True`, the service applies different AI-powered transpiling passes depending on the `optimization_level` requested.
+- `ai` ("true", "false", "auto") - Whether to use AI-powered capabilities during transpilation. The AI-powered capabilities available can be for `AIRouting` transpiling passes or other AI synthesis methods. If this value is `"true"`, the service applies different AI-powered transpiling passes depending on the `optimization_level` requested. If  `"false"`, it uses the latest Qiskit transpiling features without AI. Finally, if `"auto"`, the service decides whether to apply the standard Qiskit heuristic passes or the AI-powered passes based on your circuit.
```
Comment (abbycross) [inline docs/transpile/qiskit-transpiler-service.mdx:36]:
```suggestion
- `ai` ("true", "false", "auto") - Whether to use AI-powered capabilities during transpilation. The AI-powered capabilities available can be for `AIRouting` transpiling passes or other AI-powered synthesis methods. If this value is `"true"`, the service applies different AI-powered transpiling passes depending on the `optimization_level` requested. If  `"false"`, it uses the latest Qiskit transpiling features without AI. Finally, if `"auto"`, the service decides whether to apply the standard Qiskit heuristic passes or the AI-powered passes based on your circuit.
```

**Review by abbycross — APPROVED** [review]

**Pull request merged**